image/Dockerfile

```diff
@@ -18,17 +25,45 @@ RUN apt-get update && \
     g++ \
     libsm6 \
     zlib1g-dev \
-    libopenexr-dev && \
-  /golem/install_py_libs.sh /golem/work/requirements.txt && \
-  /golem/copy.sh && \
-  apt-get remove -y \
-    libopenexr-dev \
-    zlib1g-dev \
-    wget \
-    libxrender1 \
-    libsm6 \
-    g++ \
-    libglib2.0-0 && \
+    libopenexr-dev
+
+COPY --from=blender /blender /blender
+# Check to see that all libraries are available
+RUN ! ldd /blender/blender | grep "=> not found"
+
+COPY entrypoints/scripts/verifier_tools/requirements.txt /tmp
+RUN apt-get install -y python3-pip
+RUN pip3 install --upgrade pip
+RUN pip install -r /tmp/requirements.txt
+WORKDIR /usr/local/lib/python3.6/dist-packages
+# Check to see that all libraries are available
+# We don't need libgfortran
+RUN ! find . -name "*.so" | xargs ldd | grep -v "libgfortran-ed201abd.so.3.0.0" | grep "=> not found"
+
+RUN mkdir /golem-libs
+RUN { find . -name "*.so"; echo "/blender/blender"; } | xargs ldd | grep "=> /usr/lib/x86_64-linux-gnu" | awk '{$1=$1};1' | sort | rev | uniq -f 2 | rev | cut -f 3 -d" " | xargs cp -t /golem-libs/
```

Comment:
A comment for our future self would be nice here

Reply:
#magic